FetchRequests/Sources/JSON.swift

```diff
@@ -385,42 +375,42 @@ extension JSON: Collection {
 // MARK: - Literals
 
 extension JSON: ExpressibleByDictionaryLiteral {
-    public init(dictionaryLiteral elements: (String, Any)...) {
-        let data: [String: Any] = elements.reduce(into: [:]) { memo, element in
-            memo[element.0] = element.1
+    public init(dictionaryLiteral elements: (String, JSONConvertible)...) {
+        let data: [String: JSON] = elements.reduce(into: [:]) { memo, element in
```

Comment:
This is fundamentally a little more processing at run time, but should mostly be transient structs